### PR TITLE
Add composite demo, remove text-field wrapping demo [skip ci]

### PR DESCRIPTION
**Pull request by yuriy-fix**
Connects to #20

### CODE REVIEW

File: demo/custom-field-basic-demos.html [hunk 1]
```diff
@@ -6,16 +6,44 @@
...
+        <vaadin-custom-field label="Choose appointment time">
+          <vaadin-date-picker></vaadin-date-picker>
+          <vaadin-time-picker></vaadin-time-picker>
+        </vaadin-custom-field>
+        <script>
+          window.addDemoReadyListener('#custom-field-basic-demos-composite-example', function(document) {
+            const phoneNumber = document.querySelectorAll('vaadin-custom-field')[0];
+            const dateTime = document.querySelectorAll('vaadin-custom-field')[1];
+            const logChanges = function(e) {
+              console.warn('change event with value: ' + e.detail.value);
```
Comment (web-padawan) [inline demo/custom-field-basic-demos.html]: 1. As a developer, I would expect rather `console.info` for output of this kind

2. Please consider a way to demonstrate that value contains Tab character separator

Comment (yuriy-fix) [reply demo/custom-field-basic-demos.html]: 1. Using `console.info` now
2. Using stringified value now

File: demo/custom-field-basic-demos.html [hunk 2]
```diff
@@ -6,16 +6,44 @@
...
+                <vaadin-item>+46</vaadin-item>
+                <vaadin-item>+34</vaadin-item>
+              </vaadin-list-box>
+            </template>
+          </vaadin-select>
+          <vaadin-number-field></vaadin-number-field>
+          <vaadin-number-field></vaadin-number-field> 
         </vaadin-custom-field>
+        <br>
+        <vaadin-custom-field label="Choose appointment time">
```
[the remark this answers is not in the document]
Comment (yuriy-fix) [reply demo/custom-field-basic-demos.html]: Simplified.

File: demo/custom-field-basic-demos.html [hunk 3]
```diff
@@ -6,16 +6,44 @@
...
     </style>
 
-    <h3>Single element wrapping</h3>
-    <vaadin-demo-snippet id="custom-field-basic-demos-sample-example" when-defined="vaadin-custom-field">
+    <h3>Creating Composite of Multiple Fields</h3>
+    <vaadin-demo-snippet id="custom-field-basic-demos-composite-example" when-defined="vaadin-custom-field">
       <template preserve-content>
-        <vaadin-custom-field label="Name">
-          <vaadin-text-field></vaadin-text-field>
+        <vaadin-custom-field label="Enter your mobile phone number">
```
[the remark this answers is not in the document]
Comment (yuriy-fix) [reply demo/custom-field-basic-demos.html]: Simplified.

File: demo/custom-field-basic-demos.html [hunk 4]
```diff
@@ -6,16 +6,44 @@
       }
     </style>
 
-    <h3>Single element wrapping</h3>
-    <vaadin-demo-snippet id="custom-field-basic-demos-sample-example" when-defined="vaadin-custom-field">
+    <h3>Creating Composite of Multiple Fields</h3>
```
[the remark this answers is not in the document]
Comment (yuriy-fix) [reply demo/custom-field-basic-demos.html]: Renamed.

File: demo/custom-field-advanced-demos.html [hunk 5]
```diff
@@ -25,30 +33,15 @@ <h3>Usage with iron-form</h3>
...
-            const btn = document.querySelector('vaadin-button');
-            btn.addEventListener('click', function() {
+            const serializeBtn = document.querySelector('vaadin-button');
+
+            serializeBtn.addEventListener('click', function() {
               const serializedValues = form.serializeForm();
               document.querySelector('#msg').innerHTML =
-                'date: ' + serializedValues['invoice-date'] + '<br>' +
-                'price: ' + serializedValues['price'];
+                `date: ${serializedValues['invoice-date']}
```
[the remark this answers is not in the document]
Comment (yuriy-fix) [reply demo/custom-field-advanced-demos.html]: Good catch! Reverted.

File: demo/custom-field-advanced-demos.html [hunk 6]
```diff
@@ -16,7 +16,15 @@ <h3>Usage with iron-form</h3>
           </vaadin-custom-field>
           <vaadin-custom-field name="price" label="Price of the item">
             <vaadin-text-field></vaadin-text-field>
-            <vaadin-select></vaadin-select>
+            <vaadin-select>
```
[the remark this answers is not in the document]
Comment (yuriy-fix) [reply demo/custom-field-advanced-demos.html]: Changed.

File: README.md [hunk 7]
```diff
@@ -26,9 +26,10 @@
 ```
 -->
 ```html
-<vaadin-custom-field>
-  ...
-</vaadin-custom-field>
+<vaadin-custom-field label="Choose appointment time">
```
Comment (web-padawan) [inline README.md]: Please update the screenshot to match the updated demo
Also, "Appointment time" here is enough, too

**Pull request merged**